CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 2.7.0-rc.0 / 2019-01-18
+
+This release adds support for PromQL Subquries and disk size based retention. 
+Please note that the disk size calculations currently only account for space 
+occupied by blocks, and not the WAL (which could be of a significant size).
+
+* [FEATURE] Add Subqueries to PromQL #4831
+* [FEATURE] Add support for disk size based retention. #5109 prometheus/tsdb#343
+* [FEATURE] Add CORS origin flag. #5011
+* [ENHANCEMENT] TSDB: Run CI tests on Windows too to catch windows specific errors. prometheus/tsdb#469
```

Comment:
not sure we need this and it was already there for v2.6.0

Reply:
We don't mention things that aren't user visible, like this